setup.cfg

```diff
@@ -16,6 +16,7 @@ install_requires =
     setuptools
     pyjwt
     grpcio
+    click==8.0.4
```

Comment:
Is this customer facing pacakge? Do we use this just for testing ? Should it maybe live down in `extras_require` section with `mypy` and `black`?

Reply:
Removed that and updated the black module to the latest version based on this post: https://github.com/psf/black/issues/2964